exporter/elasticsearchexporter/model.go

```diff
@@ -247,6 +254,9 @@ func encodeLogAttributesECSMode(document *objmodel.Document, attrs pcommon.Map,
 			}
 
 			document.AddAttribute(ecsKey, v)
+			if _, ok := preserveMap[k]; ok {
```

Comment:
Since it is map[string]bool, can we rely on the fact that non-existent keys will return zero (=false) value for cleaner code?
```suggestion
			if preserve := preserveMap[k]; preserve {
```

Reply:
Good catch, thank's! Fixed with c4c5f40